src/applications/disability-benefits/686c-674-v2/config/chapters/report-add-child/index.js

```diff
@@ -0,0 +1,146 @@
+      path: '686-report-add-child/:index/information',
+      uiSchema: information.uiSchema,
+      schema: information.schema,
+    }),
+    addChildIdentification: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: "Child's Identification",
+      path: '686-report-add-child/:index/identification',
+      uiSchema: identification.uiSchema,
+      schema: identification.schema,
+    }),
+    addChildPlaceOfBirth: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: "Child's Place of Birth",
+      path: '686-report-add-child/:index/place-of-birth',
+      uiSchema: placeOfBirth.uiSchema,
+      schema: placeOfBirth.schema,
+    }),
+    addChildRelationship: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: 'Your relationship to this child',
+      path: '686-report-add-child/:index/relationship',
+      uiSchema: relationship.uiSchema,
+      schema: relationship.schema,
+    }),
+    addChildStepchild: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: "Child's biological parents",
+      path: '686-report-add-child/:index/stepchild',
+      uiSchema: stepchild.uiSchema,
+      schema: stepchild.schema,
+    }),
+    addChildAdditionalInformationPartOne: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: 'Additional information needed to add child',
+      path: '686-report-add-child/:index/additional-information-part-one',
+      uiSchema: additionalInformationPartOne.uiSchema,
+      schema: additionalInformationPartOne.schema,
+    }),
+    addChildAdditionalInformationPartTwo: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: 'Additional information needed to add child',
+      path: '686-report-add-child/:index/additional-information-part-two',
+      uiSchema: additionalInformationPartTwo.uiSchema,
+      schema: additionalInformationPartTwo.schema,
+    }),
+    addChildChildAddressPartOne: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: "Child's Address",
+      path: '686-report-add-child/:index/child-address-part-one',
+      uiSchema: childAddressPartOne.uiSchema,
+      schema: childAddressPartOne.schema,
+    }),
+    addChildChildAddressPartTwo: pages.itemPage({
+      depends: formData => isChapterFieldRequired(formData, TASK_KEYS.addChild),
+      title: "Child's Address",
+      path: '686-report-add-child/:index/child-address-part-two',
+      uiSchema: childAddressPartTwo.uiSchema,
+      schema: childAddressPartTwo.schema,
+    }),
+  };
+});
+
+export const chapter = {
+  title: 'Add one or more children',
+  pages: {
+    ...chapterPages,
+    // addChildInformation: {
```

Comment:
Did you mean to have this all commented out?

Reply:
Not present at least! I went ahead and removed the comments.